doc/tutorial/ranges/range_snippets.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <vector>
+#include <range/v3/view/drop.hpp>
+#include <seqan3/range/view/all.hpp>
+#include <seqan3/std/view/all.hpp>
+
```

Comment:
Do you need all these includes?
is `#include <seqan3/std/ranges>` enough?

Reply:
no, range/view/all includes our views, std/view/all includes the standard's views

Reply:
drop is superfluous though